rootless.md

```diff
@@ -0,0 +1,27 @@
+# Shortcomings of Rootless Podman
+
+* Not a ton of “How To” documentation or documentation in general
+* If you don’t have /etc/subuid and /etc/subgid setup for a user, then it can easily fail
+* This causes issues on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
+  * Can not create containers that bind to ports < 1024
+* No Cgroup Support (hopefully fixed when Cgroups V2 happens).
+* Can not share container images with CRI-O or other users
+* Difficult to use Additional Stores for sharing content
+* Does not work on NFS Homedirs or noexec/nodev homedirs
+* Can not use Overlayfs driver, but does support fuse-overlayfs
+  * Does not perform as well as root podman with Kernel based Overlay
+* Only other supported driver is VFS.
+* No KATA Container support ???
+* Difficult to use multiple different User Namespaces
+* No CNI Support
+* Cannot use ping (Can be fixed by setting sysctl on host)
+* Requires new shadow-utils (not found in older (RHEL7/Centos7 distros) Should be fixed in RHEL7.7 release
+* Some commands don’t work.
+  * mount/unmount (on fuse-overlay) will work if you enter the mount namespace with a tool like buildah unshare
+  * podman stats (Lack of Cgroup support)
+  * podman varlink???
+  * Checkpoint and Restore (CRIU requires root)
+  * Pause and Unpause (no freezer cgroup)
+* Some issues with higher UIDs - can cause issues with builds, etc
+  * If a UID is not mapped within the user namespace mapping for a container, then builds will not be able to put the UID in an image.
+* Doesn’t work with FreeIPA or other domain services that manage users (yet)
```

Comment:
IDK, but maybe a fluff sentence or two (at the top or maybe the bottom) noting that these issues are all under consideration to be worked on and we'd welcome help from members of the community in moving any of them forward.

Reply:
Lets take this back to the Google Docs.  Github is not efficient for this wordsmithing.